docs/upgrade-3-4.md

```diff
@@ -119,61 +119,15 @@ ls -ltr /usr/share/armadillo/application/
 
 ## 4. Config the new version
 
-### 4.1 application template
-
-This step is optional, you can try if it works without. 
-
-The application settings could have new entries so you may need to check these.
-
-```bash
-# Change the version number v4.x.y
-wget https://raw.githubusercontent.com/molgenis/molgenis-service-armadillo/v4.x.y/application.template.yml
-```
-
-To see the difference run:
-
-```bash
-diff --side-by-side /etc/armadillo/application.yml application.template.yml
-```
-
-Your output should look like output below.
-
-- Left side column is your settings.
-- Right side column is our expected settings.
-- In the middle some symbols may occur:
-  - the `&lt;` means only your settings
-  - the `&gt;` means we have a setting (probably added or options)
-  - the `|` means both have different values which happens with OICD/oauth settings for sure.
-
-```
-armadillo:                            armadillo:
-  # set this false if you DON'T want Armadillo to create/edit      # set this false if you DON'T want Armadillo to create/edit
-  docker-management-enabled: true                  docker-management-enabled: true
-....
-audit:                                  <
-  log:                                  <
-    path: /var/log/armadillo/audit.log  <
-                                        <
-....
-
-storage:                                         storage:
-  ## to change location of the data storage        ## to change location of the data storage
-  root-dir: /usr/share/armadillo/data           |  root-dir: data
-```
-
-Making changes may be a little tricky. You can backup
-
-```bash
-cp /etc/armadillo/application.yml ./
-# list files
-ls .
-```
-
-then edit
+### 4.1 application.yml
 
+Edit the application.yml:
 ```bash
 nano /etc/armadillo/application.yml
 ```
+Add the following, below `docker-management-enabled: true`:
+`docker-run-in-container: false`
+
```

Comment:
This is not needed as we fixed that in https://github.com/molgenis/molgenis-service-armadillo/commit/c289e99aeacd721d4037a7e61db57321cf22b339

Maybe we should remove `docker-management-enabled: true` from template too ... or disable both there.

Reply:
I've added this information to the top of step 1, I think it's good to give people options and be transparant :)